apps/frontend/src/locales/fr/translation.json

```
@@ -219,42 +219,52 @@
     "deleteSite": "Supprimer le site"
   },
-  "settings": {
-    "title": "Paramètres",
-    "subtitle": "Configuration de votre dashboard parapente",
-    "tabs": {
+  "settings": {
+    "title": "Paramètres",
+    "subtitle": "Configuration de votre dashboard parapente",
+    "description": "Les préférences sont appliquées immédiatement aux pages concernées et conservées dans ce navigateur.",
+    "autoSave": "Auto-save",
+    "saved": "Appliqué",
```

Biome (2.4.15)

[error] The key saved was already declared.

(lint/suspicious/noDuplicateObjectKeys)